server/dist/controllers/trip.controller.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
```

Comment:
Add the dist folder to the server .gitignore. That way compiled build files won't hit the repo.